shared/macros.jinja

```diff
@@ -585,76 +975,214 @@ ocil_clause: "{{{ sebool }}} is not enabled"
 {{%- endmacro %}}
 
 
+{{#
+    Describe how to disable an SELinux boolean.
+
+:param sebool: The SELinux boolean to disable
+:type sebool: str
+
+#}}
 {{%- macro describe_sebool_disable(sebool) %}}
     To disable the <code>{{{ sebool }}}</code> SELinux boolean, run the following command:
     <pre>$ sudo setsebool -P {{{ sebool }}} off</pre>
 {{%- endmacro %}}
 
 
+{{#
+    Describe how to enable an SELinux boolean.
+
+:param sebool: The SELinux boolean to disable
+:type sebool: str
+
+#}}
 {{%- macro describe_sebool_enable(sebool) %}}
     To enable the <code>{{{ sebool }}}</code> SELinux boolean, run the following command:
     <pre>$ sudo setsebool -P {{{ sebool }}} on</pre>
 {{%- endmacro %}}
 
 
+{{#
+    Show how to install a package with apt-get.
+
+    Example output::
+
+        apt-get install package
+
+:param package: Package to install
+:type package: str
+
+#}}
 {{%- macro apt_get_package_install(package) %}}
     $ apt-get install {{{ package }}}
 {{%- endmacro %}}
 
 
+{{#
+    Show how to remove a package with apt-get.
+
+
+    Example output::
+
+        $ apt-get remove package
+
+:param package: Package to remove
+:type package: str
+
+#}}
 {{%- macro apt_get_package_remove(package) %}}
     $ apt-get remove {{{ package }}}
 {{%- endmacro %}}
 
 
+{{#
+    Show how to install a package with dnf.
+
+    Example output::
+
+        $ sudo dnf install package
+
+:param package: Package to install
+:type package: str
+
+#}}
 {{%- macro dnf_package_install(package) %}}
     $ sudo dnf install {{{ package }}}
 {{%- endmacro %}}
 
 
+{{#
+    Show how to remove a package with dnf.
+
+    Example output::
+
+        $ sudo dnf erase remove package
+
+:param package: Package to remove
+:type package: str
+
+#}}
 {{%- macro dnf_package_remove(package) %}}
     $ sudo dnf erase {{{ package }}}
 {{%- endmacro %}}
 
 
+{{#
+    Show how to install a package with yum.
+
+    Example output::
+
+        $ sudo yum install package
+
+:param package: Package to install
+:type package: str
+
+#}}
 {{%- macro yum_package_install(package) %}}
     $ sudo yum install {{{ package }}}
 {{%- endmacro %}}
 
 
+{{#
+    Show how to remove a package with yum.
+
+    Example output::
+
+        $ sudo yum erase remove package
```

Comment:
```suggestion
        $ sudo yum remove package
```

Reply:
Do we want to change the macro to match?

Reply:
No need. We just have to remove `remove` from the example. They are basically aliases, but in the example there is a redundant option.

Reply:
Ah sorry, didn't read my mistake correctly.